src/main/java/homework_1/Main.java

```diff
@@ -0,0 +1,26 @@
+package homework_1;
+
+public class Main {
+    private static final String WARNING_MESSAGE = "Alarm!";
+    private static final String NOT_FOUND_MESSAGE = "Arguments have not been found! Please enter arguments when starting the program!";
+    private static final String INVALID_ARGUMENT = "error";
+    private static final String OUTPUT_PATTERN = "%s: %d %s\n";
+
+    public static void main(String[] args) {
+        if (args.length < 1) {
+            System.err.println(NOT_FOUND_MESSAGE);
+            return;
+        }
+        for (String s : args) {
+            if (s.equals(INVALID_ARGUMENT)) {
+                System.err.println(WARNING_MESSAGE);
```

Comment:
Doesn't work as expected
![image](https://user-images.githubusercontent.com/58872509/125207756-af5cd300-e296-11eb-99c1-edfedcdd4c0f.png)

Reply:
![image](https://user-images.githubusercontent.com/69588345/125210897-37e56e80-e2ab-11eb-89e5-8116f2608506.png)

It seems to work properly. Should I switch the app to Russian language? First I had made it in Russian but then I saw you task in Teams and changed it. The task says that the app should stop when the argument equals 'error'. 
![image](https://user-images.githubusercontent.com/69588345/125210993-b3472000-e2ab-11eb-9f7a-4dc5f650ee51.png)

Should I switch it back to Russian?

Reply:
No, in English is fine, just change System.out.err to System.out.printls, because as you can see, if I start the app via Gradle, it finishes the app first and then prints the message.